assignment6/rearranging_cars_test.py

```diff
@@ -0,0 +1,26 @@
+'''
+Created on 07 July 2017
+
+@author: Eva
+'''
+
+import unittest
+from rearranging_cars import rearrange_cars, Move
+
+class UnknownLanguageTest(unittest.TestCase):
+    def setUp(self):
+        self.initial_array = [1, 2, 0, 3]
+        self.final_array = [3, 1, 2, 0]
+        self.solution = [Move(0,2), Move(3,0), Move(1,3), Move(2,1), Move(3,2)]
```

Comment:
Well the way I think about it, if my algorithm has generated the moves of the solution, it should also lead to the final state. I think it is as important to check the moves taken as it is checking that the initial and final array are equal.